.github/workflows/on-safe-to-test-label.yml

```diff
@@ -172,27 +179,28 @@ jobs:
         run: |
           make kind-cluster-delete
       - name: Run helm test
-        if: ${{ !contains(github.event.pull_request.labels.*.name, 'chart update') }}
+        if: ${{ !contains(github.event.pull_request.labels.*.name, 'chart update') && inputs.architecture != 'arm64' }}
         run: |
           make helm-test
       - name: Copy Kind logs to S3
-        if: ${{ !contains(github.event.pull_request.labels.*.name, 'chart update') }}
+        if: ${{ !contains(github.event.pull_request.labels.*.name, 'chart update') && inputs.architecture != 'arm64' }}
```

Comment:
So I get skipping the helm test but why skip terminating the cluster and copying the kind cluster logs to s3 for ARM?

Reply:
The kind cluster is deleted in the step before, this cluster is just used for helm test

Reply:
Got it, yeah it would be nice when we re-visit the helm test if we just broke these steps out and grouped them together as a helm test workflow.